x-pack/test/api_integration/apis/kibana/stats/stats.js

```diff
@@ -30,8 +30,9 @@ export default function ({ getService }) {
           expect(isUUID(body.kibana.uuid)).to.be.ok();
         });
 
-        it('should return 401 for extended', async () => {
-          await supertestNoAuth.get('/api/stats?extended').auth(null, null).expect(401);
+        it('should return 200 for extended', async () => {
+          const { body } = await supertestNoAuth.get('/api/stats').expect(200);
+          expect(isUUID(body.kibana.uuid)).to.be.ok();
```

Comment:
I wrote a new FTR suite before discovering that there was one already (fun fact, the 401 caused by the ES client for extended stats was a "feature" not a bug). Given they don't test *exactly* the same things, I kept mine after discovering this one.

Reply:
The behavior is different and easy to miss. ++ to adding the new API integration test